dhis-2/dhis-test-web-api/src/test/java/org/hisp/dhis/webapi/controller/tracker/export/relationship/RelationshipsExportControllerTest.java

```diff
@@ -465,6 +465,96 @@ void getRelationshipsByTrackedEntity() {
     assertHasOnlyUid(to.getUid(), "trackedEntity", relationship.getObject("to"));
   }
 
+  @Test
+  void shouldNotGetRelationshipsByTrackedEntityWhenRelationshipIsDeleted() {
+    TrackedEntity to = trackedEntity();
+    Enrollment from = enrollment(to);
+    Relationship r = relationship(from, to);
+
+    r.setDeleted(true);
+    manager.update(r);
+
+    assertNoRelationships(
+        GET("/tracker/relationships?trackedEntity={tei}", to.getUid()).content(HttpStatus.OK));
```

Comment:
```suggestion
        GET("/tracker/relationships?trackedEntity={te}", to.getUid()).content(HttpStatus.OK));
```

old habit 😁 we are now using TE/te no instance anymore 😄